### PR TITLE
Feature/12372 family members rat profile/12801 unit tests

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Adding units tests for family member RAT profiles

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12801

## Screenshots
NA

### CODE REVIEW

**Review by 30mar — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/__tests__/CreateAntigenTestProfileViewModelTests.swift [hunk 1]
```diff
@@ -5,13 +5,24 @@
 import XCTest
 @testable import ENA
 
-/* EXPOSUREAPP-12801
 class CreateAntigenTestProfileViewModelTests: CWATestCase {
 	
 	func testGIVEN_EmptyAntigenTestProfile_THEN_SetValues() {
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/__tests__/CreateAntigenTestProfileViewModelTests.swift:10]: The name somehow doesn't fit anymore?
Is this case with empty profiles tested somewhere else?

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/__tests__/CreateAntigenTestProfileViewModelTests.swift [hunk 2]
```diff
@@ -39,14 +50,28 @@ class CreateAntigenTestProfileViewModelTests: CWATestCase {
...
+			firstName: "Max",
+			lastName: "Mustermann",
+			dateOfBirth: Date(timeIntervalSince1970: 390047238),
+			addressLine: "Blumenstraße 2",
+			zipCode: "43923",
+			city: "Berlin",
+			phoneNumber: "0165434563",
+			email: "sabine.schulz@gmx.com"
+		)
+		store.antigenTestProfiles = [antigenTestProfile]
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/__tests__/CreateAntigenTestProfileViewModelTests.swift:72]: Same question here as above. Is the empty case tested somewhere else?

**Review by ArturFriesen — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/__tests__/CreateAntigenTestProfileViewModelTests.swift (hunk 2 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/__tests__/CreateAntigenTestProfileViewModelTests.swift:72]: Same.

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/__tests__/CreateAntigenTestProfileViewModelTests.swift (hunk 1 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/__tests__/CreateAntigenTestProfileViewModelTests.swift:10]: Good catch, fixed.

**Pull request merged**